lib/runtime-core/src/macros.rs

```diff
@@ -1,6 +1,15 @@
+#[macro_export]
+#[cfg(feature = "debug")]
+macro_rules! debug {
+    ($fmt:expr) => (println!(concat!("wasmer-runtime(:{})::", $fmt), line!()));
+    ($fmt:expr, $($arg:tt)*) => (println!(concat!("wasmer-runtime(:{})::", $fmt, "\n"), line!(), $($arg)*));
+}
+
+#[macro_export]
+#[cfg(not(feature = "debug"))]
 macro_rules! debug {
```

Comment:
Does this create unused variable warnings when compiling without the debug feature?

Reply:
No it does not.